### ISSUE TITLE
[wfreerdp] RemoteApp double border + crash

### ISSUE THREAD

**Issue by casper** [FreeRDP/FreeRDP#7268]
Using wfreerdp.exe static build 3.0.0-dev. 
Is Remote App supposed to work on this build? 

Testing with Notepad I get a double border type window (see screenshot), and trying to resize the window causes a crash. 

`wfreerdp /u:abc /p:xyz /app:"||Notepad" /v:server`

Is there some switch missing to get this to work, or is Remote App just abandoned for the Windows binary?

![image](https://user-images.githubusercontent.com/25386/132067724-4866862c-2b06-43f9-a092-83e8168dde18.png)

**Comment by akallabeth** [comment]
@casper ok, a few questions:
1. What is `wfreerdp.exe static build 3.0.0-dev.`?
2. Did you try with our nightly builds too?

See https://github.com/FreeRDP/FreeRDP/wiki/PreBuilds for download links

**Comment by casper** [comment]
It think it was one of the nightly builds `version 3.0.0-dev (a485306c9)`. To be sure I downloaded it again from the link you gave me (shows it was built yesterday), it now says `version 3.0.0-dev (7e7394d90)`. 

It still has the same problem, except on the first run that I did the double border was gone, but subsequent runs shows it again. 

However I'm going to close this, as the reason I was using FreeRDP before (without RemoteApp, where it works fine) was it had much better performance than MSTSC. I wanted the same performance for RemoteApp too, where MSTSC is terribly laggy. Now during the past few days I managed to [tweak MSTSC](https://serverfault.com/questions/688534/poor-mouse-update-rate-on-windows-rdp-client) to get the same performance as FreeRDP, so I am switching back to that one.
